### PR TITLE
Added TypIcons font

### PR DESCRIPTION
**Pull request by 3pehrbehroozi**
- Added TypIcons font and items to SwiftIcons.swift
 - Added TypIcons.ttf font file
 - Updated sample project for TypIcons
 - Removed unused 'break' of 'switch statements' in whole project
 - Updated readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +124,8 @@ Copy all the files from `Source` folder.
 - OpenIconic.ttf
 - Stateface.ttf
 - WeatherIcons.ttf
+- WeatherIcons.ttf
```
Comment (ranesr) [inline README.md]: Can you remove this line? `WeatherIcons` is listed twice here. Thanks.

Comment (3pehrbehroozi) [reply README.md]: Oops! yes sure.
Thanks for noticing!

Comment (3pehrbehroozi) [reply README.md]: Done!

**Comment by ranesr** [comment]
Thank you @3pehrbehroozi for your contribution. Merging these changes now.

**Pull request merged**